machinery/topology.go

```diff
@@ -168,6 +168,22 @@ func (t *Topology) Objects() *collection[Object] {
 	}
 }
 
+// All returns all object nodes in the topology.
+// The list can be filtered by providing one or more filter functions.
```

Comment:
We should remove this line from the docs.

It's actually possible to filter by calling `.Items(...FilterFunc)` on a returned _collection_, but not anymore on the these _collection builders_ `Targetables()`, `Policies()`, `Objects()` and now `All()`.